src/router.tsx

```diff
@@ -31,7 +31,7 @@ interface AppRouterProps {
 }
 
 // TODO: consider redirecting from inside /order, /wallet, /auction/:nonexistent_addr to root
-const withHeaderAndFooter = (Component: React.ComponentClass | React.SFC, headerProps?: { content?: boolean, dumb?: boolean }, useFooter = true) => (compProps: any) => (
+const withHeaderAndFooter = (Component: React.ComponentClass | React.SFC, headerProps?: { content?: boolean, dumb?: boolean, noMenu?: boolean }, useFooter = true, compProps?: any) => () => (
```

Comment:
Return `compProps` to the curried function params. Otherwise all props on eg. `HomeWH` disappear into the void. Like router props.
You can use `extraProps` or something instead

Reply:
good catch